bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -1018,7 +1052,7 @@ impl<'a> CircuitInputStateRef<'a> {
                     [offset, length]
                 }
             }
-            _ => unreachable!(),
+            _ => [Word::zero(), Word::zero()],
```

Comment:
There are so many `OpcodeId` so it might not a good idea to list them all here. If there are new op codes which `last_callee_return_data_offset` and `last_callee_return_data_length` are non-zero, they should be added in this `match`. If not, it sould be fine to set these two values  to zero. Otherwise their test cases should be failed. Does it make sense to you?